docs/EXTENDED-ARCHIVES.md

```diff
@@ -0,0 +1,344 @@
+
+### `bytesToZeroOutFromTheBeginning (Number)`
+
+The number of bytes to zero out from the beginning before starting the flash
+process. This option currently only applies when using bmaps.
+
+### `recommendedDriveSize (Number)`
+
+The recommended drive size to flash this image, in bytes.
+
+The use case for this option is that while a drive might be large enough to
+contain the image, it might not be large enough to deliver a good experience
+when actual using the application or operating system contained in the image.
+
+### `bmapPath (String)`
+
+Path to a [bmap][bmap] file.
+
+### `instructionsPath (String)`
+
+Path to a markdown file showing instructions to the user for when the flash
+completes.
+
+### `supportedDevices (String[])`
+
+An array of the device slugs this image supports.
+
+### `updateUrl (String)`
+
+The url where the client can request the latest version of the image.
+
+The way this works is that you provide a generic url that redirects to the
+latest image resource. The client will follow all redirects until it finds the
+final image, and will check if the resource matches the currently selected
+image.
+
+### `etag (String)`
+
+The [HTTP Etag][etag] of the current image.
+
+This can be used to check if the image resolved by `updateUrl` is different to
+the one we have locally.
+
+### `expiryDate (String)`
+
+The timestamp to determine at which point the image may be expired. The date
+should conform to [ISO 8601][iso8601] standard.
+
+Clients may use this information to refuse flashing the image.
+
+Client
+------
+
+> A client represents the program making use of the information about the other
+  entities. This is not something publishers declare themselves.
+
+_At the moment of this writing, the only existent client is the Etcher
+application._
+
+Here's an example of a real-world client manifest for Etcher running on Mac OS
+X:
+
+```json
+{
+  "name": "etcher",
+  "version": "1.0.0-beta.14",
+  "os": "darwin",
+  "arch": "x64"
+}
+```
+
+A client may be described with any of the following properties:
+
+### `name (String)`
+
+The client name.
+
+### `version (String)`
+
+The client version.
+
+### `os (String)`
+
+The operating system running the client.
+
+The current possible values are: `aix`, `darwin`, `freebsd`, `linux`,
+`openbsd`, `sunos`, and `win32`.
+
+### `arch (String)`
+
+The client operating system architecture.
+
+The current possible values are: `arm`, `arm64`, `ia32`, `mips`, `mipsel`,
+`ppc`, `ppc64`, `s390`, `s390x`, `x32`, `x64`, and `x86`.
+
+Extensions
+----------
+
+Each of the entities described above accept an object property called
+`extensions`, which you can use to provide your own custom properties to the
+manifests, in a way that are specific to your application. It is every
+encouraged that you put those custom properties inside the `extensions`
+namespace to avoid potential clashes with other properties that may be
+introduced in the future.
```

Comment:
I guess I was asking more for a syntax-example (again, because people love copying'n'pasting), rather than a what-would-it-be-used-for example ;-)
The doc talks about both "properties" and "namespace" so I guess the intended syntax isn't very clear.